src/lib/components/viewer/RedactionToolbar.svelte

```diff
@@ -38,9 +38,11 @@ Assumes it's a child of a ViewerContext
     X_SMALL: width < remToPx(24),
   };
 
+  $: hasRedactions = $redactions.length > 0;
```

Comment:
I think derived is appropriate here as well:
```
const hasRedactions = derived(  
  redactions,  
  ($redactions) => $redactions.length > 0,  
);
```
Maybe it's overkill here? But this habit gets us more in line with runes.

Reply:
I think this is overkill for this, but it'll definitely be cleaner with runes.